code/__DEFINES/security.dm

```diff
@@ -0,0 +1,41 @@
+#define DETSCAN_CATEGORY_FINGERS "Prints"
+#define DETSCAN_CATEGORY_BLOOD "Blood"
+#define DETSCAN_CATEGORY_FIBER "Fibers"
+#define DETSCAN_CATEGORY_DRINK "Reagents"
+#define DETSCAN_CATEGORY_ACCESS "ID Access"
+
+// custom categories section
```

Comment:
`///`

Reply:
moth, that comment is not related to specifically detscan_category_notes. its just denoting where the default categories end. 
![image](https://user-images.githubusercontent.com/73374039/206471499-b618d4a8-0dbf-4709-9af0-d9663698da5d.png)

![image](https://user-images.githubusercontent.com/73374039/206471814-85a4c4da-33c9-4adc-ba8c-2d48ec1ebc76.png)

![image](https://user-images.githubusercontent.com/73374039/206472033-a04c6cc1-a07d-4f11-864f-9814d6c70e1b.png)

![image](https://user-images.githubusercontent.com/73374039/206472383-286f90c1-8d2d-4d1e-8412-a4157ac1812e.png)
![image](https://user-images.githubusercontent.com/73374039/206472497-98392193-393f-443d-bfc8-5c28fa8ceb3f.png)

![image](https://user-images.githubusercontent.com/73374039/206472710-3da4e890-7d5a-4d75-b632-9e03d33cf591.png)

![image](https://user-images.githubusercontent.com/73374039/206473016-c7a7a135-b544-4d3b-99f3-cf4a68d50363.png)

i truly do not get why this change is necessary

Reply:
oh- sorry i was confused abt this